examples/api200/unmanaged_device.rb

```diff
@@ -11,5 +11,35 @@
 
 require_relative '../_client' # Gives access to @client
 
+def print_device(device)
```

Comment:
No need for this method here, use the simple put when necessary to avoid increasing the complexity of the example.